src/components/container/PartnerBanner/PartnerBanner.scss

```diff
@@ -0,0 +1,35 @@
+.PartnerBanner {
+    width: fit-content;
+    height: fit-content;
+    background-color: var(--color-bunting);
+
+    div {
+        display: flex;
+        flex-direction: row;
+        gap: 4vw;
+        animation: slide 10s 1s linear infinite;
+    }
+
+    img {
+        width: 10vw;
+        height: 5vw;
+        object-fit: contain;
+        left: 0%;
+        top: 0%;
+        margin: 1vw;
+    }
+}
+
+@keyframes slide {
+    0% {
+        margin-left: 0%;
+    }
+
+    50% {
+        margin-left: -25%;
+    }
+
+    100% {
+        margin-left: -50%;
+    }
+}
```

Comment:
is it a desired behaviour that the  slider will be empty for a while a soon as the last banner is moved to the left? I hope I explained it more or less ok))

Reply:
it half empty because  i wanted to show last logo properly.

Reply:
If I reproduce the code correctly,  we have increased/decreased margin size, but no slide effect.
I though we want smth like this https://codesandbox.io/s/distracted-bas-bzekny?file=/src/styles.css
However it is not working smoothly so far... Need to research a bit

Reply:
tbh I was confused about how it worked too. but it loop all the logos. now i think y i worked bcz i saw the bottom scrollbar XD thanks!